src/sentry/workflow_engine/actions/notification_action/notification_action_handler.py

```diff
@@ -0,0 +1,57 @@
+from sentry.models.group import GroupEvent
+from sentry.workflow_engine.actions.action_handlers_registry import register_action_handler
+from sentry.workflow_engine.actions.notification_action.logic import invoke_issue_alert_registry
+from sentry.workflow_engine.models.action import Action
+from sentry.workflow_engine.models.data_source import DataSource
+from sentry.workflow_engine.models.data_source_detector import DataSourceDetector
+from sentry.workflow_engine.models.detector import Detector
+
+
+@register_action_handler(
+    [
+        Action.Type.NOTIFICATION_SLACK,
+        Action.Type.NOTIFICATION_DISCORD,
+        Action.Type.NOTIFICATION_MSTEAMS,
+        Action.Type.NOTIFICATION_PAGERDUTY,
+        Action.Type.NOTIFICATION_OPSGENIE,
+        Action.Type.NOTIFICATION_GITHUB,
+        Action.Type.NOTIFICATION_GITHUB_ENTERPRISE,
+        Action.Type.NOTIFICATION_GITLAB,
+        Action.Type.NOTIFICATION_JIRA,
+        Action.Type.NOTIFICATION_JIRA_SERVER,
+        Action.Type.NOTIFICATION_AZURE_DEVOPS,
+        Action.Type.NOTIFICATION_SENTRY_APP,
+        Action.Type.NOTIFICATION_EMAIL,
+    ]
+)
+def notification_action_handler(action: Action, group_event: GroupEvent):
```

Comment:
this is the entrypoint of NOWA

Reply:
i am having trouble with this registry registering in 1 of my tests, if i import it it claims i registered this function twice. if i don't, it says this isn't registered 😕